scripts/deploy/2-check-roles.ts

```diff
@@ -20,9 +20,9 @@ const sleepTime = 3000;
 
 const main = async () => {
   let ownerAddress = socketOwner;
-  let executorAddress = executorAddresses[mode];
-  let transmitterAddress = transmitterAddresses[mode];
-  let watcherAddress = watcherAddresses[mode];
+  let executorAddress = executorAddresses[mode]; // TODO: should probably be socketOwner on Kinto?
```

Comment:
Socket team to review this 🙏🏻

Reply:
These addresses are only used for signatures. They dont need to send any transactions. So we can let them be as is.